Intersect.Client/MonoGame/Graphics/MonoRenderer.cs

```diff
@@ -426,18 +428,13 @@ public override GameRenderTexture CreateRenderTexture(int width, int height)
             Color borderColor = null
         )
         {
-            if (gameFont == null)
-            {
-                return;
-            }
-
-            var font = (SpriteFont) gameFont.GetFont();
+            var font = (SpriteFont)gameFont?.GetFont();
             if (font == null)
             {
                 return;
             }
 
-            StartSpritebatch(mCurrentView, GameBlendModes.None, null, renderTexture, false, null);
```

Comment:
Add back the `, false, null`

Reply:
![imagen](https://user-images.githubusercontent.com/17498701/210425573-127a7290-e2de-447b-99a0-e15694a07f68.png)

![imagen](https://user-images.githubusercontent.com/17498701/210425585-216abafe-18ad-4be8-a4b7-c825a2d8fd78.png)


( reverting them back )